django/river/common/analyzer/analyzer.py

```diff
@@ -211,11 +231,13 @@ def get_analysis_columns(analysis):
 
                 for condition in input_group.conditions:
                     analysis_columns.add(condition.sql_column)
+        for filter in analysis.filters:
+            analysis_columns.add(filter.sql_column)
```

Comment:
does this fix something ?

Reply:
This method is used in tests to see what's in the mapping, it was interesting to add the info of the columns that are used in filters